src/nodeinfo/NodeInfo.cpp

```diff
@@ -11,6 +11,10 @@
 
 #include "NodeInfo.h"
 
+#undef max
+#undef min
```

Comment:
Haha, where's this coming from? I have only seen this in WIN API so far.

Reply:
Ze Arduino API. There's all kind of [weird magic](https://github.com/arduino/ArduinoCore-API/blob/master/api/Common.h#L124-L149) done over there.